app/js/components/validation.js

```diff
@@ -85,8 +85,30 @@ if (form.length) {
     });
 }
 
-// Add a stricter url validator, this one ensures the protocol is set on the input value
+// Add a stricter url validator, fields validated with urlstrict are
+// allowed to be empty, or a URL with protocol.
 window.Parsley.addValidator('urlstrict', function (value, requirement) {
+    return validateEmpty(value) || validateUrl(value);
+}, 32).addMessage('en', 'urlstrict', 'This value should be a valid URL.');
```

Comment:
The validation message for the URL is probably good enough. When looking at it in a very puristic manner. The message implies I should be able to fill all sorts of URL's like: `file:///home/someuser/somefile.txt` or `ftp://foo@example.com`. Adding a hint on the allowed schemes (http/https) would take away this very unlikely confusion.